src/fdc3/dotnet/DesktopAgent/test/MorganStanley.ComposeUI.DesktopAgent.Tests/Channels/UserChannelTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Channels;
+using MorganStanley.ComposeUI.Messaging.Abstractions;
+
+namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Tests.Channels;
+
+public class UserChannelTests : ChannelTestFixture
+{
+    private const string TestChannel = "testChannel";
+    private readonly UserChannel _channel = new UserChannel(TestChannel, new Mock<IMessagingService>().Object, null);
+    private readonly  ChannelTopics _topics = Fdc3Topic.UserChannel(TestChannel);
+
+    public UserChannelTests()
+    {
+        Channel = _channel;
```

Comment:
This seems a bit odd, why not just create the UserChannel and the ChannelTopics here instead of creating a field that's never used?